packages/ticket_sidebar/src/App.tsx

```diff
@@ -92,7 +92,7 @@ const App = () => {
   const handleOrderIdChange = (orderId: string) => setOrderId(orderId)
 
   const handleExistingClientOnClick = (channel: string) => {
-    if (channel === 'chat' || channel === 'web') {
+    if (channel === 'chat' || channel === 'web' || channel === 'api') {
```

Comment:
And for me the list can be a constant like "ALLOWED_CHANNELS"
```suggestion
    if (['chat', 'web', 'api'].includes(channel)) {
```

Reply:
Definitely, thanks!